docs/_layouts/plain.html

```diff
@@ -64,3 +64,9 @@ <h1>{{ page.title }}{% if page.is_beta %} <span class="beta">Beta</span>{% endif
 {% endif %}
 
 {{ content }}
+
+{% if page.language == "zh" %}
+  <div class="footer">
+    <a href="https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications" target="_blank">想参与贡献翻译？</a>
```

Comment:
I like the idea of add this link in this page. One suggestion is can we add this link for EN version so that everyone can know this info in every language version. (just a suggestion) :)